### PR TITLE
bugfix: Don't change packages in files if they move outside workspace

### PR DESCRIPTION
**Pull request by tgodzik**
Connected to https://github.com/scalameta/metals/issues/4888

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala [hunk 1]
```diff
@@ -2103,10 +2103,14 @@ class MetalsLspService(
   ): CompletableFuture[WorkspaceEdit] =
     CancelTokens.future { _ =>
       val moves = params.getFiles.asScala.toSeq.map { rename =>
-        packageProvider.willMovePath(
-          rename.getOldUri().toAbsolutePath,
-          rename.getNewUri().toAbsolutePath,
-        )
+        val oldPath = rename.getOldUri().toAbsolutePath
+        val newPath = rename.getNewUri().toAbsolutePath
+        if (newPath.startWith(workspace))
```
Comment (kpodsiad) [inline metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala:2108]: How is this related with moving outside of workspace?

Comment (kpodsiad) [inline metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala:2108]: Add comment ;)

**Review by kpodsiad — APPROVED** [review]

File: metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala (hunk 1 above)
Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala:2111]: Good idea, added.
